src/Lorentz/Contracts/BaseDAO/Proposal.hs

```diff
@@ -267,7 +267,7 @@ submitVote = do
 
   constructT @(Proposal pm)
     ( fieldCtor $ do
-        -- | Update upvote
+        -- Update upvote
```

Comment:
Hm, if it's invalid Haddock and CI doesn't check it, I think we should add a CI check to prevent invalid Haddock in future. See https://gitlab.com/morley-framework/morley/-/merge_requests/620/diffs for example.